keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -612,6 +677,8 @@ enabled even if the annotation has been set on the Service.
 
 - KEP Merged: February 2021
 - Alpha release: Kubernetes 1.21
+- Beta Release: Kubernetes 1.23
+- Feature Gate on-by default, feature available by default: 1.24
```

Comment:
Normally beta implies "feature gate on by default"... did we just declare beta in 1.23 but leave it disabled? Was that an accident or by design? Might be worth explaining just for clarity...

Reply:
Definitely an accident, and realized a few days too late :(. Will add a note.